handbook/product/roadmap.md

```diff
@@ -74,7 +74,28 @@ See [Distribution roadmap](https://github.com/sourcegraph/about/pull/1104).
 
 ## Search
 
-TO DO
+1. 🔄 Enterprise homepage
+1. 🔄 Search tour
+1. 🔄 Search expressions
+1. 🔄 Scaling indexed text search to 500k repositories
+1. 🔄 Streaming search
+1. Code monitoring (private code monitors, no sharing)
```

Comment:
Does this include webhooks or only email for the initial release? Can we put webhooks as a separate item in this list if we think it doesn't need to exist in the initial release?

Reply:
@poojaj-tech @christinaforney should we have a separate roadmap item here?

Reply:
Stage 1 of code monitoring as described in [code monitoring planning](https://docs.google.com/document/d/1EG42dM1old49uPXpGAqL7b9k95bWpM0Zcu0JM5gXtOQ/edit) includes webhooks, so let's just make them explicit in this roadmap item?

```suggestion
1. Code monitoring (private code monitors, no sharing, emails + webhooks)
```

Reply:
Sounds good!